README.md

```diff
@@ -20,6 +20,15 @@ createdb projectlab
 createdb projectlab_test
 ```
 
+For Windows you can install postgres from [here](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads) and you can follow [this](https://www.prisma.io/dataguide/postgresql/setting-up-a-local-postgresql-database#setting-up-postgresql-on-windows) easy setup (it also works for MacOS)
+
+Then open the SQL Shell and write this commands for creating a DB:
+
+```
+CREATE DATABASE projectlab;
+CREATE DATABASE projectlab_test;
+```
```

Comment:
Should we add a command to create a user with a password? I'm guessing when installing postgres you set up a password for the postgres user, but maybe it would be a good practice to create a user specifically for project lab, with a password, that we can later use in the connection string, instead of using the postgres user in the connection string for projectlab.